src/sagemaker_containers/_modules.py

```diff
@@ -171,7 +171,7 @@ def download_and_install(uri, name=DEFAULT_MODULE_NAME, cache=True):
             install(module_path)
 
 
-def run(module_name, args=None, env_vars=None, wait=True):  # type: (str, list, dict, bool) -> Popen
+def run(module_name_or_script, args=None, env_vars=None, wait=True, bash=False):  # type: (str, list, dict, bool) -> Popen
```

Comment:
bash is a bad name. script could be any interpreter on the host, controlled by #! line

Reply:
also, we don't need this args at all. why not just branch on file extension (py vs sh or py means python anything else means sh)

if that won't work because of what we are doing with module name before calling this, then we could require .sh extension, or refactor the caller so it what it is.